packages/openactive-broker-microservice/app.js

```diff
@@ -1,29 +1,29 @@
-const express = require("express");
-const logger = require("morgan");
-const request = require("request");
-const axios = require("axios");
-const nSQL = require("@nano-sql/core").nSQL;
-const moment = require("moment");
-const config = require("config");
-const criteria = require("@openactive/test-interface-criteria").criteria;
+/* eslint-disable no-console */
+/* eslint-disable no-use-before-define */
```

Comment:
I think this is a good rule, but fair, one for another day!

Reply:
Yes agreed, it would have messed with the rest of the diff so good to introduce in a single PR